command/helper/config.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	Join           string                 `json:"join_addr"`
 	Consensus      map[string]interface{} `json:"consensus"`
 	RestoreFile    string                 `json:"restore_file"`
+	BlockTime      uint64                 `json:"block_time"`
```

Comment:
I would note somewhere that this is the time in milliseconds, just so we're clear when looking at this configuration

Reply:
Comment added in 631515404cf370d1fc015938ea35da52ff9dbaaf